libs/langchain/langchain/agents/agent_iterator.py

```diff
@@ -190,14 +190,14 @@ def __iter__(self: AgentExecutorIterator) -> Iterator[AddableDict]:
             name=self.run_name,
         )
         try:
-            while self.agent_executor._should_continue(
+            while self.agent_executor._should_continue(  # noqa: SLF001
```

Comment:
It's OK for `AgentExecutorIterator` to access private members of `AgentExecutorIterator`